src/main/java/vazkii/botania/api/package-info.java

```diff
@@ -1,4 +1,4 @@
-@API(owner = "Botania", apiVersion = "76", provides = "BotaniaAPI")
+@API(owner = "Botania", apiVersion = "77", provides = "BotaniaAPI")
```

Comment:
Don't need to bump the API version as a release has not been made since it was increased to 76